### PR TITLE
libstrophe 0.12.1

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/libstrophe.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Libstrophe < Formula
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "check"
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
```
[the remark this answers is not in the document]
Comment (Porkepix) [reply Formula/libstrophe.rb:23]: Yes, while I check and it's already compatible with OpenSSL 3.0, the reverse dependency tree is causing issues on several formulas.

**Review by chenrui333 — APPROVED** [review]

File: Formula/libstrophe.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/libstrophe.rb]: Yeah, I think it can be addressed after [this PR](https://github.com/Homebrew/homebrew-core/pull/85775)

**Comment by mhmdanas** [comment]
Is the `check` dependency actually required? The formula builds just fine for me without it.

**Comment by carlocab** [comment]
> Is the `check` dependency actually required? The formula builds just fine and passes the test for me without it.

Looks like they removed it quite a few years ago. https://github.com/strophe/libstrophe/commit/f0706f56c45561b399d0519787c8018a4860fa3e

File: Formula/libstrophe.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Libstrophe < Formula
   desc "XMPP library for C"
   homepage "https://strophe.im/libstrophe/"
-  url "https://github.com/strophe/libstrophe/archive/0.12.0.tar.gz"
-  sha256 "f645819d8d93711c92454974dd9007c9b9e98ea0b59cb708dc626dd4c6b9d0a8"
+  url "https://github.com/strophe/libstrophe/archive/0.12.1.tar.gz"
+  sha256 "91fc40a89528a32b0bda975d84f8f567b57e687898e22e328b3733c2ae0393d8"
   license any_of: ["GPL-3.0", "MIT"]
```
Comment (carlocab) [inline Formula/libstrophe.rb:6]:
```suggestion
  license all_of: ["GPL-3.0-only", "MIT"]
```
https://metadata.ftp-master.debian.org/changelogs//main/libs/libstrophe/libstrophe_0.12.1-1_copyright

**Review by carlocab — COMMENTED** [review]
We should fix the license and remove the unnecessary dependency in a separate PR.

**Pull request closed without merging**